File: test_login.py
import httpx

# Test with correct test credentials
with httpx.Client(base_url="http://localhost:8001") as client:
    res = client.post("/api/auth/login", json={
        "email": "admin-a@test.local",
        "password": "TestPass123!"
    })
    print(f"Direct Backend - Status: {res.status_code}")
    print(f"Direct Backend - Body: {res.text[:500]}")

# Test through Next.js proxy
with httpx.Client(base_url="http://localhost:3000") as client:
    res = client.post("/api/auth/login", json={
        "email": "admin-a@test.local",
        "password": "TestPass123!"
    })
    print(f"\nVia Next.js - Status: {res.status_code}")
    print(f"Via Next.js - Body: {res.text[:500]}")

File: apps/web/test-login.js
async function testLogin() {
    try {
        const res = await fetch('http://localhost:3000/api/auth/login', {
            method: 'POST',
            headers: { 'Content-Type': 'application/json' },
            body: JSON.stringify({
                email: 'admin-a@test.local',
                password: 'TestPass123!'
            })
        });
        
        console.log('Status:', res.status);
        console.log('Set-Cookie:', res.headers.get('set-cookie'));
        
        const text = await res.text();
        console.log('Body:', text);
    } catch (e) {
        console.error(e);
    }
}

testLogin();
